helm/reana/values.yaml

```diff
@@ -20,6 +20,10 @@ workspaces:
   paths:
     - /var/reana:/var/reana
 
+interactive_sessions:
+  cronjob_schedule: "0 3 * * *"  # everyday at 3am
+  maximum_duration: forever
```

Comment:
I think this might be a little bit misleading, as an administrator might think that this is the maximum duration of an interactive session since the time it was started. What about naming it `maximum_inactivity_period` or something similar to this?

The description of the value in README.md and in CHANGES.rst would also need to be changed